packages/c/sshnpd/src/run_srv_process.c

```diff
@@ -12,96 +12,40 @@
 
 #define LOGGER_TAG "RUN SRV"
 
-/*
- * Notes about this particular code:
- * This code has some history. I originally implemented this code to call an external srv binary using the exec family
- * of sys call functions. Thus instead of calling the functions from srv directly, we build the list of args and pass
- * it to the srv arg parsing function. Ideally we refactor this later, but it works for now :)
- */
```

Comment:
These changes are the planned refactors mentioned in the comment. We went from generating a string to pass to the srv over the commandline, to using the srv structs and calling its functions internally as a single binary application.